yb-voyager/src/utils/utils.go

```diff
@@ -130,6 +130,29 @@ func GetSchemaObjectList(sourceDBType string) []string {
 	return requiredList
 }
 
+func ApplyExportSchemaObjectListFilter(allowedObjects *[]string, expectedObjects string) {
+	if expectedObjects == "" {
+		return
+	}
+	expectedObjectsSlice := strings.Split(expectedObjects, ",")
+	newAllowedObjects := []string{}
+	for _, object := range *allowedObjects {
+		if ContainsString(expectedObjectsSlice, object) {
+			newAllowedObjects = append(newAllowedObjects, object)
+		}
+	}
+	*allowedObjects = newAllowedObjects
+}
+
+func ContainsString(list []string, str string) bool {
+	for _, object := range list {
+		if strings.EqualFold(object, str) {
+			return true
+		}
+	}
+	return false
+}
```

Comment:
We can get rid of this by converting the object-type names provided by the user to all upper case.